Guides/Development.md

```diff
@@ -116,7 +120,6 @@ If you have a setup for developing the driver in an editor other than the ones l
 ## Workflow
 1. Create a feature branch, named by the corresponding JIRA ticket if exists, along with a short descriptor of the work: for example, `SWIFT-30/writeconcern`.
 1. Do your work on the branch.
-1. If you add, remove, or rename any tests, make sure to update `LinuxMain.swift` accordingly. If you are on MacOS, you can do that by running `make linuxmain`.
```

Comment:
Is this still relevant? I dont see `make linuxmain`

Reply:
nope! we used to have maintain a list of tests because Swift could not auto-discover tests on Linux, however that is no longer necessary.